.github/CODEOWNERS

```diff
@@ -1 +1,3 @@
-* @iNikem @mateuszrzeszutek @signalfx/gdi-instrumentation-java
+* @signalfx/gdi-java-maintainers @signalfx/gdi-java-approvers
+
+CODEOWNERS @signalfx/gdi-java-maintainers
```

Comment:
does this work without `.github/CODEOWNERS`?

Reply:
Yes -- apparently it will match all files named `CODEOWNERS` in this repo, regardless of location.